app/views/activities/order_screen.html.erb

```diff
@@ -176,29 +191,32 @@
               Sumup betaling
             </span>
           </a>
-          <button type="button" class="btn btn-dark" v-bind:disabled="orderConfirmButtonDisabled()"
+          <button type="button" class="btn btn-dark" :disabled="orderConfirmButtonDisabled()"
                   @click="confirmOrder">
             <%= fa_icon 'money' %>
           </button>
         </div>
 
-        <button @click="confirmOrder" class="btn btn-block btn-lg btn-success"
-                :class="{ 'btn-warning' : (orderRequiresAge() && selectedUser && selectedUser.minor) }"
-                :disabled="orderConfirmButtonDisabled()"
-                v-if="!payWithPin || !<%= @sumup_enabled.to_s %>">
-          <span class="text-uppercase">
-            Bestelling plaatsen
+        <button @click="maybeConfirmOrder" class="btn btn-block btn-lg btn-success"
+              :class="{ 'btn-warning' : selectedUser && ((orderRequiresAge() && selectedUser.minor) || selectedUser.insufficient_credit) }"
+              :disabled="orderConfirmButtonDisabled()"
```

Comment:
Does this update reactively? Reading this makes me feel like it doesnt, since it performs a method call and doesn't use a computes property

Reply:
Yes, it works with Vue and whenever an order is placed an updated user object is returned from the server and updated in the user list on the client side.

Reply:
The functions were already there and they worked previously so apparently it does :)

Reply:
A function is apparently run with every rerender while a computed property is only recomputed if a used attribute updates, so it is more efficient. I moved the above functions to computed properties.